core/stdlib/std.ncl

```diff
@@ -934,6 +934,27 @@
         "%
       = fun pred label value => if pred value then value else %blame% label,
 
+    Sequence
+      | doc m%"
+          Apply multiple contracts from left to right.
```

Comment:
Might be worth slightly more documentation, such as why do we need that if it's equivalent to just apply several contracts with `x | Foo | Bar`. A simple example with `Array (std.contract.sequence [Foo, Bar])` that can't be expressed currently with simple `|`s could be sufficient.